driver_openal.go

```diff
@@ -12,14 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build darwin freebsd
+// +build darwin freebsd openbsd
 // +build !js
 // +build !android
 
 package oto
 
 // #cgo darwin  LDFLAGS: -framework OpenAL
 // #cgo freebsd LDFLAGS: -lopenal
+// #cgo openbsd LDFLAGS: -L/usr/local/lib/ -lopenal
+// #cgo openbsd CFLAGS: -I/usr/local/include/
```

Comment:
Is adding `/usr/local/lib` mandatory? On FreeBSD, I expect users to add such values to environment variables.